internal/cli/auth/login_test.go

```diff
@@ -77,9 +77,10 @@ func Test_loginOpts_Run(t *testing.T) {
 	buf := new(bytes.Buffer)
 
 	opts := &LoginOpts{
-		flow:       mockFlow,
-		config:     mockConfig,
-		NoBrowser:  true,
+		flow:      mockFlow,
+		config:    mockConfig,
+		NoBrowser: true,
+		// todo: do not test using SkipConfig flag (deprecated)
```

Comment:
Any ticket for the todo? Let's include

Reply:
task was created but good point about including it here, thanks.
done in this PR: https://github.com/mongodb/mongodb-atlas-cli/pull/1218/files